samples/nrf5340/empty_app_core/prj.conf

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+CONFIG_MULTITHREADING=n
+CONFIG_NUM_PREEMPT_PRIORITIES=0
+CONFIG_SYS_CLOCK_EXISTS=n
+CONFIG_ARM_MPU=n
+CONFIG_SIZE_OPTIMIZATIONS=y
+CONFIG_I2C=n
+CONFIG_WATCHDOG=n
+CONFIG_GPIO=n
+CONFIG_PINMUX=n
+CONFIG_SPI=n
+CONFIG_SERIAL=n
+CONFIG_FLASH=n
+CONFIG_SYS_POWER_MANAGEMENT=n
+CONFIG_DYNAMIC_INTERRUPTS=n
+CONFIG_IRQ_OFFLOAD=n
+CONFIG_THREAD_STACK_INFO=n
+CONFIG_THREAD_CUSTOM_DATA=n
+CONFIG_FLOAT=n
```

Comment:
Not really needed, since this is =n by default, (and it has also been renamed)

Reply:
I rebased  the commit to newest master and I still see the old name, but as you noticed it is `n` by default, so I can delete it.

Reply:
It is renamed to `config FPU`, it will come with the current upmerge.

Reply:
> It is renamed to `config FPU`, it will come with the current upmerge.

Yes, I am aware of it, but it is not yet there, so I must use the old name.
It doesn't matter now, because as you pointed, it is `n` by default, so I will delete it.